signature-crate/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name          = "signature"
 description   = "Traits for cryptographic signature algorithms (e.g. ECDSA, Ed25519)"
-version       = "0.3.0" # Also update html_root_url in lib.rs when bumping this
+version       = "1.0.0-pre" # Also update html_root_url in lib.rs when bumping this
```

Comment:
Note: if we do this, I'd like to do a series of prereleases (e.g. `1.0.0-pre.0`, `1.0.0-pre.1`) to work through issues encountered with `ed25519-dalek`, `signatory`, and potentially `rsa` as well /cc @roblabla